src/compiler/crystal/command/docs.cr

```diff
@@ -103,9 +115,27 @@ class Crystal::Command
     compiler.wants_doc = true
     result = compiler.top_level_semantic sources
 
-    Doc::Generator.new(result.program, included_dirs, output_directory, output_format, sitemap_base_url, sitemap_priority, sitemap_changefreq).run
+    Doc::Generator.new(result.program, included_dirs, output_directory, output_format, sitemap_base_url, sitemap_priority, sitemap_changefreq, project_info).run
 
     report_warnings result
     exit 1 if warnings_fail_on_exit?(result)
   end
+
+  private def create_project_info(name, version)
+    name ||= Doc::ProjectInfo.find_default_name
+    unless name
```

Comment:
What about just... Not including the little "project summary" box if there's no name.